elasticsearch5/src/main/java/org/vertexium/elasticsearch5/ElasticsearchSearchQueryBase.java

```diff
@@ -756,6 +756,9 @@ protected QueryBuilder getFilterForTextPredicate(TextPredicate compare, HasValue
                     throw new VertexiumException("Unexpected text predicate " + has.predicate);
             }
         }
+        if (compare.equals(TextPredicate.DOES_NOT_CONTAIN)) {
```

Comment:
I swear this is the change @sfeng88 made earlier

Reply:
It looks like it was only made in the ES 1.7 module in the previous PR.

Reply:
The tests definitely passed for me last night...does the ES5 test have a new profile? that might be it.